spec/json/matchers/match_response_schema_spec.rb

```diff
@@ -15,8 +15,8 @@
 
   it "fails when the body is missing a required property" do
     create_schema("foo_schema", {
-      type: "object",
-      required: ["foo"],
+      "type" => "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.